gbrl/common/utils.py

```
@@ -251,15 +251,21 @@ def setup_optimizer(optimizer: Dict, prefix: str = '') -> Dict:
     assert isinstance(lr, (int, float, str)), "lr must be a float or string"
     if isinstance(lr, str) and 'lin_' in lr:
         assert 'T' in optimizer, "Linear optimizer must contain T the total"
         "   number of iterations used for scheduling"
         lr = lr.replace('lin_', '')
         optimizer['scheduler'] = 'Linear'
+    # Normalize scheduler name (linear -> Linear, const -> Const)
+    sched = optimizer.get('scheduler', 'Const').lower()
+    optimizer['scheduler'] = 'Linear' if sched == 'linear' else 'Const'
     optimizer['init_lr'] = float(lr)
+    if optimizer['init_lr'] <= 0:
+        raise ValueError("init_lr must be > 0")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)